sound/smooth_swing_v2.h

```diff
@@ -94,38 +105,66 @@ class SmoothSwingV2 : public SaberBasePassThrough {
         if (speed < smooth_swing_config.SwingStrengthThreshold) {
 #if 1
           if (monitor.ShouldPrint(Monitoring::MonitorSwings)) {
-            STDOUT.print("speed: ");
-            STDOUT.println(speed);
+           STDOUT.print("speed: ");
+           STDOUT.println(speed);
           }
 #endif
           break;
         }
         state_ = SwingState::ON;
         
       case SwingState::ON:
+        //check for AccentSwingThreshold, presence of accent swings and if the accent player is stopped (this prevents clipping)
+        if (speed >=smooth_swing_config.AccentSwingSpeedThreshold
+          && accent_swings_present
+            && (A.player->isPlaying() || B.player->isPlaying())
+        {
+          //allocate player
+          if(!accent_player_) {
+            accent_player_ = GetFreeWavPlayer();
+          }
+          else {
```

Comment:
This shouldn't be an else.
Instead we need to check if (accent_player_) {
As it is, the first time we get here, we'll allocate the accent player, but not start playing.

Reply:
ok, resolved.